package.json

```diff
@@ -128,7 +128,7 @@
     "chai-arrays": "2.2.0",
     "chai-as-promised": "7.1.1",
     "cheerio": "1.0.0-rc.12",
-    "chokidar-cli": "latest",
```

Comment:
This should be `*` (what it is on master), not `latest`.

Reply:
Doh...Missed that!